test/libraries/Position.t.sol

```diff
@@ -21,4 +22,41 @@ contract PositionTest is Test {
         }
         assertEq(positionSlot, expectedPositionSlot, "slots not equal");
     }
+
+    function test_fuzz_update(
+        int128 liquidityDelta,
+        uint256 feeGrowthInside0X128,
+        uint256 feeGrowthInside1X128,
+        uint128 liquidity
+    ) public {
+        // grab a position and set its values
+        Position.Info storage position = positions[0];
+        position.liquidity = liquidity;
+        position.feeGrowthInside0LastX128 = feeGrowthInside0X128;
+        position.feeGrowthInside1LastX128 = feeGrowthInside1X128;
+        if (position.liquidity == 0 && liquidityDelta == 0) {
+            vm.expectRevert(Position.CannotUpdateEmptyPosition.selector);
+        }
+        uint128 oldLiquidity = position.liquidity;
```

Comment:
these `oldX` feel unnecessary? arent they just equal to the parameters of this test?